TCMSApp/src/client/app/runs/runs.controller.js

```diff
@@ -13,12 +13,19 @@
         .module('app.runs')
         .controller('RunsController', RunsController);
 
-    RunsController.$inject = ['logger', 'FakeRunsFactory'];
+    RunsController.$inject = ['logger', 'FakeRunsFactory', 'moment'];
 
-    function RunsController(logger, FakeRunsFactory) {
+    function RunsController(logger, FakeRuns, moment) {
 
         var vm = this;
```

Comment:
maybe something like runsVm would be better. 
Why? So that we can access parent controller by its name from any child controller